Lombiq.Privacy/Startup.cs

```diff
@@ -43,6 +43,20 @@ public override void ConfigureServices(IServiceCollection services)
             options.MinimumSameSitePolicy = SameSiteMode.Strict;
             options.Secure = CookieSecurePolicy.Always;
             options.ConsentCookie.Expiration = new TimeSpan(365, 0, 0, 0);
+
+            options.OnAppendCookie = cookieContext =>
```

Comment:
Now one can't register via Azure AD, and the cookie banner doesn't get saved (but just on that page):

https://github.com/Lombiq/Orchard-Privacy/assets/1976647/8e56fd58-4642-437f-b6d6-e4413bd9e64b